util/util.go

```diff
@@ -3,20 +3,3 @@
 // Licensed under the BSD-3-Clause License (the "License").
 // You may not use this file except in compliance with the License.
 package util
-
```

Comment:
This file does not contain code anymore, it should be removed.

Reply:
There are other files like this; sure.